### PR TITLE
[SC-151] Refactor Windows AMI

### PR DESCRIPTION
**Pull request by zaro0508**
* Remove creation of Admin password
* Reset template and scripts with contents from packer windows
  example[1]
* Update source image to the latest version of the
  Windows_Server-2019-English-Full-Base-2020.09.09 AMI

[1] https://www.packer.io/docs/builders/amazon/ebs#connecting-to-windows-instances-using-winrm

### CODE REVIEW

File: src/template.json [hunk 1]
```diff
@@ -20,8 +21,10 @@
         "SourceAmiName": "{{ .SourceAMIName }}"
       },
       "type": "amazon-ebs",
-      "user_data_file": "./bootstrap_win.txt",
-      "winrm_password": "SuperS3cr3t!!!!",
```
Comment (tthyer) [inline src/template.json:24]: I read in the packer doc you linked in the description "You'll notice that this config does not define a user or password; instead, Packer will ask AWS to provide a random password that it generates automatically." Which sounds like an improvement, but how do you get access to the password then? Does it show up in the log when packer creates the image? What's the process for keeping track of it then, put in LP?

File: src/template.json [hunk 2]
```diff
@@ -34,41 +37,21 @@
...
-      "type": "windows-restart"
-    },
-    {
-      "environment_vars": [
-        "VAR1=A$Dollar",
-        "VAR2=A`Backtick",
-        "VAR3=A'SingleQuote",
-        "VAR4=A\"DoubleQuote"
+        "C:/ProgramData/Amazon/EC2-Windows/Launch/Scripts/InitializeInstance.ps1 -Schedule",
+        "C:/ProgramData/Amazon/EC2-Windows/Launch/Scripts/SysprepInstance.ps1 -NoShutdown"
```
Comment (tthyer) [inline src/template.json:42]: I don't see anything wrong with this, but it is unclear why this change is happening, can you spell it out?

**Review by tthyer — APPROVED** [review]
Looks ok, just had a couple questions

File: src/template.json (hunk 1 above)
Comment (zaro0508) [reply src/template.json:24]: OK.  Just to be clear there are two groups of users in windows so let's discuss them separately.

Administrator - you get the admin password by provisioning an instance with this AMI then you goto EC2-> select the instance -> Actions ->Connect -> provide the imagecentral ssh private key (get it from lastpass) -> click "Get Password" button.  Now you can connect to it with Admin user.

End User - provision an instance with this AMI thru the SC and we have things in the SC that associate the provisioned instance with jumpcloud.  The under user uses their JC credentials to login.

File: src/template.json (hunk 2 above)
Comment (zaro0508) [reply src/template.json:42]: The things remove are not needed.  i believe @thomasyu888 was still experimenting and testing so he left that cruft around. As for  InitializeInstance.ps1 and SysprepInstance.ps1.  Those were in the packer examples and i believe the reason why you need them is that it will allow all new Instances launched off this AMI to decrypt the password from your AWS Console. here's some more info about it: https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2launch.html#Schedule-EC2Launch-to-run-on-every-boot

File: src/template.json (hunk 1 above)
Comment (tthyer) [reply src/template.json:24]: 👍

File: src/template.json (hunk 2 above)
Comment (tthyer) [reply src/template.json:42]: 👍

**Review by tthyer — APPROVED** [review]

File: src/template.json (hunk 1 above)
Comment (ahayden) [reply src/template.json:24]: you can also connect to the instance through SSM via your IAM user, in which case it will not need the Admin user's password... you can set it to whatever value you want after connecting too.

Comment (zaro0508) [reply src/template.json:24]: Ahh cool! @ahayden is correct, I verified that you can also connect with Actions -> Connect -> Session Manager
